sale_resource_booking/wizards/resource_booking_sale.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from odoo.tests.common import Form
+
+
+class ResourceBookingSale(models.TransientModel):
+    _name = "resource.booking.sale"
+    _description = "Sale order generator for resource booking types"
+
+    type_id = fields.Many2one(
+        "resource.booking.type",
+        string="Booking type",
+        index=True,
+        ondelete="cascade",
+        required=True,
+    )
+    partner_id = fields.Many2one(
+        "res.partner", string="Customer", index=True, ondelete="cascade", required=True
```

Comment:
Why add an index to a TransientModel? I think this doesn’t provide any benefit, since users never search on this field in the model.

```suggestion
        "res.partner", string="Customer", ondelete="cascade", required=True
```